drivers/wifi/nrf700x/zephyr/inc/zephyr_fmac_main.h

```diff
@@ -29,7 +29,7 @@
 #include <fmac_api.h>
 #include <host_rpu_umac_if.h>
 
-#define NRF700X_DRIVER_VERSION "1.2.4.0"
+#define NRF700X_DRIVER_VERSION "1.2.4.99"
```

Comment:
Instead of manually managing this, we can just use `NCS_VERSION_STRING` or `STRINGIFY(NCS_VERSION_STRING)`

Reply:
Done.